### PR TITLE
Fix build.py path to find Windows Python.

### PR DESCRIPTION
**Pull request by dekimir**
Replace --cmake and --ninja with --path.

On Cygwin, escape to native Windows.

### CODE REVIEW

**Comment by dekimir** [comment]
@dneto0 @Qining @antiagainst @AWoloszyn PTAL.

File: utils/build.py [hunk 1]
```diff
@@ -137,31 +111,38 @@ def main():
...
-                        help='Path to the cmake executable. Default is %s.'
-                        % cmake_default)
-
-    ninja_default = os.path.join('prebuilts', 'ninja', arch, 'ninja')
-    parser.add_argument('--ninja', dest='ninja',
-                        default=os.path.join(os.getcwd(), ninja_default),
-                        help='Path to the ninja executable. Default is %s.'
-                        % ninja_default)
+    path_default = (os.path.join(os.getcwd(), 'prebuilts', 'cmake', arch, 'bin')
+                    + os.pathsep +
```
Comment (dneto0) [inline utils/build.py]: I'd write this as os.pathsep.join(  .... list .... )

But this is FINE.

**Comment by dneto0** [comment]
+2   Provided the corrsponding manifest is updated to get the Windows python.
Good to go.

**Comment by dekimir** [comment]
Pushed into master via 7eda6bf.

**Pull request closed without merging**